teamwork/templates/projects/create_project.html

```diff
@@ -65,6 +59,18 @@
           page: params.page
         };
       },
+       
+  // $(".s2_desired_techs").select2({
```

Comment:
Is the select 2 function not setup for technologies?

Reply:
its in templates/projects/edit_project.html

Reply:
Yes, but is it not a part of project creation like skills?

Reply:
We've put in s2 function for technologies in the same create_project.html before but it gave us an error. We have the select 2 function for technologies in edit_project.html

Reply:
Okay, what was the error? Just cause something is causing an error doesn't mean it should just be avoided

Reply:
When we put in the s2 function for technologies in the create_project, the technologies that we add are not being displayed in the ui

Reply:
Ooops our bad, it worked, we must have missed it last time!